angular/projects/spark-angular/src/lib/components/sprk-masthead/sprk-masthead.stories.ts

```diff
@@ -231,6 +231,107 @@ export const defaultStory = () => ({
         </a>
       </div>
     </sprk-masthead>
+    <div class="sprk-u-pam">
+      <p class="sprk-u-mbm">
+        Molestie convallis gravida neque fermentum
```

Comment:
Hey @RVMendoza, these paragraphs are not part of the masthead component and having them there is a bit confusing to users. They show up in the copy code block too, so they are copying a wall of text that isn't related to the component.

Reply:
There may be some better way, but I feel having content here best represents the masthead component and all of it's features.

One of the things I was thinking about when making masthead is that I couldn't test whether or not scrolling features worked on small viewport. I added this content to be able to test that.

Then I removed it.

Later, I had to test it one more time and had to add the content again.

I understand the sb is for devs to test and work on, as well as an implementer.

If I take this out, it feels weird to me that a user cannot test to see what masthead does on small viewport scrolling. It also feels weird that as a developer, anytime I need full access to everything masthead does, I HAVE to add fake content before I develop.

Maybe we can add this as a knob later on? Body content knob?

Reply:
thanks for removing these @RVMendoza !